docs/yml.md

```diff
@@ -29,15 +29,44 @@ image: a4bc65fd
 
 ### build
 
-Path to a directory containing a Dockerfile. When the value supplied is a 
-relative path, it is interpreted as relative to the location of the yml file 
-itself. This directory is also the build context that is sent to the Docker daemon.
+Path to a context containing a Dockerfile. When the value supplied is a relative filesystem
+path, it is interpreted as relative to the location of the config file.
+
+The value supplied to this parameter can have any of the forms accepted by the docker remote
+api ```/build``` endpoint. Namely:
```

Comment:
Single back-ticks are sufficient for inline code in MarkDown (use triple back-ticks (or 4 spaces indentation) for code "blocks")